### PR TITLE
31 create oih polling trigger

### PR DESCRIPTION
**Pull request by ElBabbler**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # 1.1.0 (November 27, 2019)
 ## Actions
     * Add Delete Object action
+    * Add Get new abd updated files trigger
```
Comment (kirill-levitskiy) [inline CHANGELOG.md]: What **abd** means?

File: spec-integration/pollingIntegration.spec.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    this.on('error', (value) => this.error.push(value));
+    this.on('end', () => {
+      this.end += 1;
+    });
+  }
+}
+
+// eslint-disable-next-line func-names
+describe('SFTP integration test - polling', function () {
+  this.timeout(20000000);
```
Comment (kirill-levitskiy) [inline spec-integration/pollingIntegration.spec.js]: timeout for 5.5 hours?

File: component.json [hunk 3]
```diff
@@ -165,6 +165,62 @@
           }
         }
       }
+    },
+    "pollingTrigger": {
+      "main": "./lib/triggers/read.js",
+      "title": "Get new abd updated files",
```
Comment (shulkaolka) [inline component.json]:
```suggestion
      "title": "Get new and updated files",
```

File: component.json [hunk 4]
```diff
@@ -165,6 +165,62 @@
           }
         }
       }
+    },
+    "pollingTrigger": {
+      "main": "./lib/triggers/read.js",
```
Comment (shulkaolka) [inline component.json]:
```suggestion
      "main": "./lib/triggers/polling.js",
```

**Pull request merged**